src/components/preferences/PreferencesPage.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import styles from "./styles";
+import PreferencesTab from "./PreferencesTab";
+import ShortcutsTab from "./ShortcutsTab";
+import { DialogActions, Button } from "@material-ui/core";
+import { Formik, Form } from "formik";
+import { DEConfirmationDialog, LoadingMask } from "@cyverse-de/ui-lib";
+
+const useStyles = makeStyles(styles);
+
+export default function PreferencesPage(props) {
+    const [restoreDef, setRestoreDef] = useState(false);
+    const classes = useStyles();
+    const { config } = props;
+
+    const handleSubmit = (values, actions) => {
+        actions.setSubmitting = true;
+        console.log(values);
+    };
+
+    const restoreDefaults = (setFieldValue) => (event) => {
+        setFieldValue("preferences.rememberLastPath", true);
+        setFieldValue("preferences.saveSession", true);
+        setFieldValue("preferences.enableImportEmailNotification", true);
+        setFieldValue("preferences.enableWaitTimeMessage", true);
+        setFieldValue("preferences.enableAnalysisEmailNotification", true);
+        setFieldValue("preferences.enableHPCPrompt", true);
+        setFieldValue("preferences.notificationKBShortcut", "N");
+        setFieldValue("preferences.dataKBShortcut", "D");
+        setFieldValue("preferences.closeKBShortcut", "Q");
+        setFieldValue("preferences.appsKBShortcut", "A");
+        setFieldValue("preferences.analysisKBShortcut", "Y");
+        // TODO find a way to get users default output folder
+        //setFieldValue("preferences.default_output_folder", )
```

Comment:
Haven't yet reviewed the rest but wanted to quickly comment on this - I totally forgot to update you after the config file was updated 😖 Sorry

So the config file that gets mounted in has been updated, you can see the QA config file [here](https://gitlab.cyverse.org/core-sw/k8s-resources/blob/master/resources/configs/qa/sonora.yaml).  You'll see it's a yaml file and the value you need is in `irods.home_path`.

So, now you need to update the `configuration.js` file in Sonora to parse that yaml file and your new irods home path value.  You'll see we're first validating that the values exist: https://github.com/cyverse-de/sonora/blob/596193a16d0ff2ef01fd0c1683409dfd5392d216/src/server/configuration.js#L83-L87

And then, after validated, we're exporting the values from the yaml file out of this configuration file for usage: https://github.com/cyverse-de/sonora/blob/596193a16d0ff2ef01fd0c1683409dfd5392d216/src/server/configuration.js#L202-L220

You just need to do the same thing for your `irods.home_path` value.

Once you have that change, then you can use the exported value from `configuration.js` here in your preferences component to build the user's home path. Part of the user's home path is their cyverse ID, which you can get using the `useUserProfile` hook that we built into Sonora.  [DataNavigation](https://github.com/cyverse-de/sonora/blob/596193a16d0ff2ef01fd0c1683409dfd5392d216/src/components/data/DataNavigation.js#L353) has a quick example.